lib/primitive/lib/Cardano/Wallet/Primitive/Ledger/Shelley.hs

```diff
@@ -1018,21 +1021,22 @@ rewardAccountFromAddress (W.Address bytes) = refToAccount . ref =<< parseAddr by
     refToAccount (SL.StakeRefPtr _) = Nothing
     refToAccount SL.StakeRefNull = Nothing
 
+newtype UnsealedTxException = UnsealedTxInUnsupportedEra AnyCardanoEra
```

Comment:
**Suggestion:**

Since this exception pertains to the `unsealTx` operation, and since it is only returned if it is **_not_** possible to unseal a transaction:

```suggestion
newtype UnsealTxException = UnsealTxInUnsupportedEra AnyCardanoEra
```